README.md

```diff
@@ -149,6 +149,7 @@ Then, update the projects to include this additional file:
 * [CI Infrastructure](https://github.com/SonarSource/re-ci-infrastructure)
 * [CI Infrastructure docs](https://github.com/SonarSource/re-ci-infrastructure/blob/master/languages-dotnet/README.md)
 * [VM Images repository](https://github.com/SonarSource/re-ci-images)

```

Comment:
@gregory-paidis-sonarsource is this still relevant? Are we still using stuff from there or is it from `dotnet-infra` only?

"clean as you document"

Reply:
or https://github.com/SonarSource/re-ci-infrastructure/blob/master/languages-dotnet/README.md .. it seems to point to GCP anyway, that's old

Reply:
Yes, the `re-ci-images` is still used.

Observe the `ctrl+f dotnet` hits inside `ec2_images` directory:
![image](https://github.com/SonarSource/sonar-dotnet/assets/115458417/fea12cec-3e2a-4ec5-b922-8177b646adda)

Here lies the configuration/build of the images we use for peach (the ones with peach on their name), the sonar-dotnet+scanner image (the one with no `base` on the name) and the sonar-security c# frontend image (the `...-base-...` one).

The idea is that:
- we use [packer](https://www.packer.io/) to build the images and add all our malware (vs-build tools, java, etc) inside `re-ci-images`. For us, this repo answers questions like "which version of X do we install". 
- we provision a pool using [CDK](https://en.wikipedia.org/wiki/AWS_Cloud_Development_Kit) (in the olden days this was GCP with [terraform](https://www.terraform.io/) I think) in which agents instantiate the aforementioned images^ inside `dotnet-infra` to actually run our pipelines. For us, this repo answers like "how many vCPUs, ram, disk space do the VMs have access to".

I am not sure if we care for `re-ci-infrastructure` anymore. 
I see some terraform files in there that are 2 years old, I am going to ask Mate if we use anything from there.